diff --git a/node_modules/wpapi/lib/util/named-group-regexp.js b/node_modules/wpapi/lib/util/named-group-regexp.js
index 0a927dd..0ae2ae9 100644
--- a/node_modules/wpapi/lib/util/named-group-regexp.js
+++ b/node_modules/wpapi/lib/util/named-group-regexp.js
@@ -14,10 +14,10 @@ const pattern = [
 	'[>\']',
 	// Get everything up to the end of the capture group: this is the RegExp used
 	// when matching URLs to this route, which we can use for validation purposes.
-	'([^\\)]*)',
+	'([^\\)]*(\\))?)\\??',
 	// Capture group end
 	'\\)',
-].join( '' );
+].join('');
 
 module.exports = {
 	/**
@@ -35,5 +35,5 @@ module.exports = {
 	 *
 	 * @prop {RegExp} namedGroupRE
 	 */
-	namedGroupRE: new RegExp( pattern ),
+	namedGroupRE: new RegExp(pattern),
 };
